tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+import logging
+import matplotlib.pyplot as plt
+import pandas as pd
+from os.path import dirname
+
+import hail as hl
+import hail.expr.aggregators as agg
+
+from gnomad.utils.reference_genome import get_reference_genome
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s", 
+    datefmt="%m/%d/%Y %I:%M:%S %p", 
+)
+logger = logging.getLogger("sex_check")
+logger.setLevel(logging.INFO) 
+
+
+def get_chr_cov(mt: hl.MatrixTable, build: str, chr_name: str, af_field: str = "AF", call_rate_threshold: float = 0.25, af_threshold: float = 0.01) -> hl.Table:
```

Comment:
```suggestion
def get_chr_cov(mt: hl.MatrixTable, build: str, chr_name: str, af_field: str = "AF", call_rate_threshold: float = 0.25, af_threshold: float = 0.01) -> hl.expr.Float64Expression:
```
this function no longer returns a hail Table; it returns an expression